<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COwnAircraftComponent</class>
 <widget class="QFrame" name="COwnAircraftComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Own aircraft</string>
  </property>
  <layout class="QGridLayout" name="gl_OwnAircraftComponent" columnstretch="0,0,0,0,0,0,0,0,0">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="1" column="2">
    <widget class="QCheckBox" name="cb_Model">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="lbl_AircraftCombinedType">
     <property name="toolTip">
      <string>Aircraft type e.g. L2J, L1P, ....</string>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QLineEdit" name="le_Callsign">
     <property name="toolTip">
      <string>e.g. DEMBZ</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhUppercaseOnly</set>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
     <property name="placeholderText">
      <string>e.g. DEMBZ</string>
     </property>
    </widget>
   </item>
   <item row="6" column="6" colspan="2">
    <widget class="BlackGui::Components::CDbAirlineIcaoSelectorComponent" name="selector_AirlineIcao">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QLabel" name="lbl_AircraftIcaoDesignator">
     <property name="toolTip">
      <string>ICAO code e.g.  B737, A320, F18</string>
     </property>
     <property name="text">
      <string>Aircraft</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="6">
    <widget class="BlackGui::Components::CAircraftModelStringCompleter" name="comp_ModelStringCompleter">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="7">
    <widget class="QWidget" name="wi_ModelString" native="true">
     <layout class="QHBoxLayout" name="hl_ModelString">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="le_SimulatorModel">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>unique model identifier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_SimulatorLookup">
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>refresh</string>
        </property>
        <property name="icon">
         <iconset resource="../../blackmisc/blackmisc.qrc">
          <normaloff>:/diagona/icons/diagona/icons/arrow-circle-225.png</normaloff>:/diagona/icons/diagona/icons/arrow-circle-225.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="6" colspan="2">
    <widget class="BlackGui::Components::CDbAircraftIcaoSelectorComponent" name="selector_AircraftIcao">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QLineEdit" name="le_AircraftCombinedType">
     <property name="toolTip">
      <string>e.g. L2J</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="placeholderText">
      <string>e.g. &quot;L2J&quot;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="lbl_LiverySend">
     <property name="text">
      <string>Send livery</string>
     </property>
    </widget>
   </item>
   <item row="6" column="8">
    <widget class="BlackGui::CTickLabel" name="lblp_AirlineIcao">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="BlackGui::CTickLabel" name="lblp_Callsign">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="5">
    <widget class="QLineEdit" name="le_SendLivery">
     <property name="placeholderText">
      <string>livery string send</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lbl_Model">
     <property name="toolTip">
      <string>Launch </string>
     </property>
     <property name="text">
      <string>Model</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="lbl_Callsign">
     <property name="text">
      <string>Callsign</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lbl_ModelSent">
     <property name="text">
      <string>Send model</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="BlackGui::CTickLabel" name="lblp_AircraftCombinedType">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QLabel" name="lbl_AircraftIcaoAirline">
     <property name="toolTip">
      <string>ICAO code e.g. DLH, LHA, ...</string>
     </property>
     <property name="text">
      <string>Airline</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="cb_Livery">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="8">
    <widget class="BlackGui::CTickLabel" name="lblp_AircraftIcao">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="8">
    <widget class="QPushButton" name="pb_Clear">
     <property name="toolTip">
      <string>clear livery</string>
     </property>
     <property name="icon">
      <iconset resource="../../blackmisc/blackmisc.qrc">
       <normaloff>:/diagona/icons/diagona/icons/cross.png</normaloff>:/diagona/icons/diagona/icons/cross.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::CTickLabel</class>
   <extends>QLabel</extends>
   <header>blackgui/ticklabel.h</header>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbAircraftIcaoSelectorComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbaircrafticaoselectorcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbAirlineIcaoSelectorComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbairlineicaoselectorcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CAircraftModelStringCompleter</class>
   <extends>QFrame</extends>
   <header>blackgui/components/aircraftmodelstringcompleter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_SimulatorModel</tabstop>
  <tabstop>pb_SimulatorLookup</tabstop>
  <tabstop>cb_Model</tabstop>
  <tabstop>comp_ModelStringCompleter</tabstop>
  <tabstop>cb_Livery</tabstop>
  <tabstop>le_SendLivery</tabstop>
  <tabstop>pb_Clear</tabstop>
  <tabstop>le_Callsign</tabstop>
  <tabstop>selector_AircraftIcao</tabstop>
  <tabstop>le_AircraftCombinedType</tabstop>
  <tabstop>selector_AirlineIcao</tabstop>
 </tabstops>
 <resources>
  <include location="../../blackmisc/blackmisc.qrc"/>
 </resources>
 <connections/>
</ui>
